docker-compose.yaml

```diff
@@ -27,6 +27,8 @@ services:
       POSTGRES_USERNAME: ${POSTGRES_SCHEMA_USERNAME:-postgres}
       AZURE_STORAGE_ACCOUNT_NAME: ${AZURE_STORAGE_ACCOUNT_NAME:-devstoreaccount1}
       AZURE_STORAGE_USE_CONNECTION_STRING: "true"
+      FFC_API_PATH: ffcApi
+      STATEMENT_RECEIVER_ENDPOINT: http://ffc-pay-statement-receiver
```

Comment:
If this is intended to use the Kubernetes traffic then should be `http://ffc-pay-statement-receiver.ffc-pay-<ENVIRONMENT>`

But I don't think this should use the Kubernetes network because the requests will be coming from outside the cluster from CRM, so this value will result in an unusable link being created.

The traffic route will be `CRM -> App Gateway -> AKS -> NGINX -> ffc-pay-statement-receiver`.  So the link needs to be for a route we'll setup in App Gateway.

Not sure if you've added something in App Config to override these values.  We could maybe do something clever in the helm chart to build them depending on the `environmentCode`.

For path, same suggestion as above about naming.  GDS also has some standards we could factor in here -> https://www.gov.uk/guidance/gds-api-technical-and-data-standards

I'd also suggest we should include a version number too, eg `/v1/` so we can extend the API in future safely.